content/guidedLearning/guideForUnityDevs.md

```diff
@@ -0,0 +1,83 @@
+
+# A very brief history of Babylon
+
+Looking into the 'Babylonian' history, this project was thought up by two Microsoft employees (David Catuhe and David Rousset) and originally released in 2013. Meaning that, unofficially, this project is backed by Microsoft. It's not just a Microsoft party though, there's loads of other dedicated community members contributing as well. It's been made as a web 3D engine, but more recently they've made a native side to it. This is an actual native implementation, with almost the same javascript/typescript interface as its web counterpart. Which means, before you tackle a big native project, you can just roughly test out your idea on the online playground (https://playground.babylonjs.com/) and copy over that code into the native project.
+Currently, there's three repositories to keep an eye on:
+
+- https://github.com/BabylonJS/Babylon.js
+- https://github.com/BabylonJS/BabylonNative
+- https://github.com/BabylonJS/BabylonReactNative
+
+# Why Babylon is amazing
+
+So, to me, Babylon is wild. It's all open-source and maintained by some very talented people. The documentation is great (it can lag a bit behind the actual implementations though), the responses on the forum are just unreasonably quick (https://forum.babylonjs.com/), and I've been lucky enough that the developers will take time to answer any and all questions I've had about Babylon React Native. Compare that to the current state of Unity, where you can still build great stuff with it, but let's be honest, Unity is a big untouchable impersonal monolith. I don't feel like the developers working for Unity have a lot of personal investment in the work they do. And that's inevitable, but the charm of Babylon is that it's not this big monolith. It's a bunch of people invested in the work they do, happy to get you started with it.
+
+Also, it's free. Basically no strings attached (make sure to check the licenses though), you can just straight up use it. So to come back to my point at the beginning of this paragraph. Babylon is wild. But where do you begin?
+
+# Where do you begin
+
+This was honestly a lot harder than I think it should be. I've found that there's a lot of prior knowledge assumed with every tutorial. First of all assuming you have a MacBook without actually stating that. The reason being that if you want to release iOS apps, you have to have some kind of Apple computer. If you're trying this on a windows computer, you may run into issues.
+
+So to get you up to speed enough for you to have some grasp on the concepts. Here's what I think is the most important:
+
+- NPM/Node.js/NPX/NVM
+- node_modules
+- package.json and package-lock.json
+- The `android` and `ios` folders
```

Comment:
I feel like there is a gap here. Are we looking at some specific example repo or assuming the reader has already followed some instructions from BabylonReactNative repo?